picasso/src/main/java/com/squareup/picasso3/PicassoExecutorService.java

```diff
@@ -59,7 +59,8 @@ public int compareTo(PicassoFutureTask other) {
 
       // High-priority requests are "lesser" so they are sorted to the front.
       // Equal priorities are sorted by sequence number to provide FIFO ordering.
-      return (p1 == p2 ? hunter.sequence - other.hunter.sequence : p2.ordinal() - p1.ordinal());
+      return (p1 == p2
+        ? hunter.getSequence() - other.hunter.getSequence() : p2.ordinal() - p1.ordinal());
```

Comment:
Can you use `@JvmField` on all these properties so that code like this doesn't have to change?